docs/installation/installation.md

```diff
@@ -16,6 +16,12 @@ Before running the operator
 * A secret in the namespace of your operator deployment telling the operator how to communicate with Tyk
 * The CRDs must be registered with the Kubernetes apiserver
 * cert-manager must be installed
+* If you are using `pro` edition make sure in your gateway's `tyk.conf` `policies.allow_explicit_policy_id` is set to true
+```json
+    "policies": {
+        "allow_explicit_policy_id": true
+    },
+```
```

Comment:
This is great. I feel like in k8s environment, an env var would more likely be used.

Reply:
We don't deploy tyk so it's up to our customers to decide how the set the value.